clang/include/clang/Basic/Builtins.td

```diff
@@ -253,8 +253,10 @@ def FmodF16F128 : F16F128MathTemplate, Builtin {
 
 def FrexpF16F128 : F16F128MathTemplate, Builtin {
   let Spellings = ["__builtin_frexp"];
-  let Attributes = [FunctionWithBuiltinPrefix, NoThrow];
+  let Attributes = [FunctionWithBuiltinPrefix, NoThrow, Const, ConstexprSince];
```

Comment:
The `frexp` family of functions cannot be marked `const` because they modify the object pointed to by the pointer parameter.

Reply:
Done.